terms-generators.lua

```diff
@@ -478,10 +474,11 @@ local function define_multi_enum(flex, flex_name, fn_replace, fn_specify, fn_uni
 		return ok, table.unpack(fn_unify(table.pack(...)))
 	end
 
+	local flex_name_dot = flex_name .. "."
```

Comment:
Why bother making a whole variable for this single trivial subexpression? This makes the code less readable for no benefit. Don't do this.